charts/osm/values.schema.json

```diff
@@ -1384,6 +1447,30 @@
                         }
                     },
                     "additionalProperties": false
+                },
+                "preinstall": {
+                    "$id": "#/properties/osm/properties/preinstall",
+                    "type": "object",
+                    "title": "The preinstall schema",
+                    "description": "Preinstall configurations",
+                    "required": [],
+                    "properties": {
```

Comment:
Let's also add `"additionalProperties": false` to these to match everything else.